frontend/src/pages/NewRun.tsx

```diff
@@ -1130,12 +1139,13 @@ export class NewRun extends Page<{ namespace?: string }, NewRunState> {
   private _getCloneName(oldName: string): string {
     const numberRegex = /Clone(?: \(([0-9]*)\))? of (.*)/;
     const match = oldName.match(numberRegex);
+    const { t } = this.props;
     if (!match) {
       // No match, add Clone prefix
-      return 'Clone of ' + oldName;
+      return `${t('common:cloneOf')} ` + oldName;
     } else {
       const cloneNumber = match[1] ? +match[1] : 1;
-      return `Clone (${cloneNumber + 1}) of ${match[2]}`;
+      return `${t('common:clone')} (${cloneNumber + 1}) ${t('common:of')} ${match[2]}`;
```

Comment:
This might happen in other places then here, but there should be a way in i18next to set make in one label, and insert the values in it. Might be https://www.i18next.com/translation-function/nesting#passing-options-to-nestings or https://www.i18next.com/translation-function/interpolation